dataquality/core/init.py

```diff
@@ -113,6 +114,36 @@ def _set_labels_for_existing_run(project_name: str, run_name: str) -> None:
         return
 
 
+def create_run_name(project_name: str) -> str:
+    """Creates an auto-incrementing run_name for a given project
+
+    If a run_name is not passed into `init`, we create a run_name base with today's
+    date, and increment the digit at the end based on how many runs were created in
+    this project with this scheme.
+
+    ie:
+        2023-05-15-1
+        2023-05-15-2
+        2023-05-15-3
+        ...
+        2023-05-15-n
```

Comment:
Looks like we are using `-` as the delimiter. In which case, the docstring is incorrect...
```suggestion
    ie:
        2023-05-15_1
        2023-05-15_2
        2023-05-15_3
        ...
        2023-05-15_n
```

Reply:
Nice catch. I changed it halfway through, missed the docstring